src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -534,43 +534,26 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var view = obj["View"];
             if (view != null)
             {
-                var nodesView = view["NodeViews"];
-                if (nodesView != null)
+                var nodeViews = view["NodeViews"].ToList();
+                foreach (var node in nodeViews)
```

Comment:
can we call this `var node view` - I find it confusing to call both the model and view node.